apps/ideas/views.py

```diff
@@ -299,3 +300,21 @@ class IdeaListView(filter_views.FilteredListView):
     def get_queryset(self):
         queryset = super().get_queryset().annotate_comment_count()
         return queryset
+
+    def get_context_data(self, **kwargs):
```

Comment:
Do I have to put that into the context or can I get the phase in the list view somehow?

Reply:
You could factor out the phase stuff into a property:

```python
@property
def active_phase(self):
     return Phase.objects.active_phases().last()
```
And than use it in your function below:

```python
if self.active_phase:
  data = self.active_phase.content().get_phase_filters(self.active_phase.module.project.pk)
```

From the template that property can be accessed as:
```
{{ view.active_phase.title }}
```